talent/hiring_process.md

```diff
@@ -42,44 +46,36 @@ This document explains the internal hiring process at source{d} and the responsi
 
 - Lead engineers are responsible to make the calls as it's a motivating factor for the developers. Leads will also be able to assess early in the process if the interview process should continue or not.
 
-### Coding Challenges
+### Coding Challenge
 
-- Every developer who passes the first Skype interview should complete the coding challenge.
-
-- In case of the Machine Learning team, there is an additional ML task necessary to be solved.
+- Check out the [Coding Challenge section in the application process](application-process.md#coding-challenge) guide.
 
 - Coding challenges are being sent to the developers by the Talent Manager and the results are being reviewed by the lead engineers.
 
-- Coding challenge is currently available in Go, Python, JavaScript, Ruby, Scala and Java. Other languages can be requested and we will consider the possibility of adding them.
+- We send coding challenges with a scheduled email. A template is available on Lever. [Boomerang for GMail](https://www.boomeranggmail.com/) can be used to schedule.
```

Comment:
I looks like this will be available directly on gmail now https://gsuiteupdates.googleblog.com/2019/04/write-now-send-later-with-schedule-send.html

Reply:
April's Fool?

Reply:
Haha, could be. But gmail was originally announced on April 1st, it must be an internal joke to announce on April Fools. And it has an entry in their help center.